config/beam.yaml

```diff
@@ -0,0 +1,6 @@
+--- 
+default_branch: master
+name: apache_beam
+repository: incubator-beam
+scm_type: github
+verification_command: "mvn clean verify"
```

Comment:
I think (if I'm reading this right) we should use "mvn clean install", because we archetypes are only validated during an install?